.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!--
+Thanks for your contribution!
+
+Please ensure that any applicable requirements below are satisfied before submitting this pull request. This will help ensure a quick and efficient review cycle.
+-->
+
+**Description**
+
+_What type of change have you made?_
+
+- BREAKING:
+  - _Append BREAKING: to your PR title_
+  - _Describe the how to patch your code for the breaking changes._
+  - _Create PRs for other projects with patches to breaking changes._
+- FIXED:
+  - _Append FIXED: to your PR title_
+  - _Describe the change you have made_
+- CHANGED:
+  - _Append CHANGED: to your PR title_
+  - _Describe the change you have made_
+- ADDED:
+  - _Append ADDED: to your PR title_
+  - _Describe the change you have made_
+- DEPRECATED:
+  - _Append DEPRECATED: to your PR title_
+  - _Describe the change you have made_
+- SECURITY:
+  - _Append SECURITY: to your PR title_
+  - _Describe the change you have made_
+
+**Checklist**
+
+- [ ] There is a related GitHub issue
```

Comment:
We haven't generally required that GitHub issues be opened for each PR. Especially not for internal contributors. Not sure it's useful to have this here if we aren't requiring it.

Reply:
This checklist is something that we have had on the mobile side but I see your point with that item. I will remove it.